Source/WebCore/editing/cocoa/EditorCocoa.mm

```diff
@@ -289,8 +289,8 @@
     types.append(String(legacyStringPasteboardType()));
     auto context = PagePasteboardContext::create(m_document.pageID());
     ALLOW_DEPRECATED_DECLARATIONS_BEGIN
```

Comment:
Can remove the `ALLOW_DEPRECATED_DECLARATIONS_BEGIN` and `ALLOW_DEPRECATED_DECLARATIONS_END` now that we're no longer using the deprecated constants, here and everywhere else changes were made.